### DISCUSSION TITLE
Move app question to top of help view?

### DISCUSSION THREAD

**Discussion by drewvolz** [StoDevX/AAO-React-Native#5549] [Ideas]
I feel that it should take precedence so that it isn’t missed.

**Comment by hawkrives** [comment]
I like having wifi reporting at the top, personally

**Comment by drewvolz** [comment]
I’ll play around with layouts in the near future.

**Comment by drewvolz** [comment]
Thinking about a grid of buttons and icons

**Comment by hawkrives** [comment]
Or a two-column Pinterest-like thing with less text + smaller / bigger / changed buttons…?

**Comment by hannesmcman** [comment]
This would be one way of resolving #2827. I would also suggest we change the title of that section to “Have an app question? Found a bug?”

**Comment by hannesmcman** [comment]
@hawkrives
